sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -26,16 +26,17 @@ export interface BeginRecoverDeletedKeyOptions extends KeyPollerOptions {
 
 // @public
 export interface CreateEcKeyOptions extends CreateKeyOptions {
-    curve?: KeyCurveName;
-    hsm?: boolean;
+    curve?: KeyCurveName; // Should we rename to curveName and deprecate curve?
+    hsm?: boolean; // Should we rename to hardwareProtected and deprecate hsm (breaking change)?
+    // Do we need Name?
 }
 
 // @public
 export interface CreateKeyOptions extends coreHttp.OperationOptions {
     enabled?: boolean;
     readonly expiresOn?: Date;
     keyOps?: KeyOperation[];
-    keySize?: number;
+    keySize?: number; // Not in dotnet
```

Comment:
Because it's our base class and not all key types would take a key size (they do not, but others in the future may not). Since our Create*KeyOptions classes derive from that, only those that take a `key_size` define the property.